.github/workflows/local-testnet.yml

```diff
@@ -69,6 +69,14 @@ jobs:
         run: ./stop_local_testnet.sh local-blinded
         working-directory: scripts/local_testnet
 
+      - name: Start a local testnet and run assertoor tests
+        uses: ethpandaops/kurtosis-assertoor-github-action@v1
+        with:
+          ethereum_package_args: scripts/tests/assertoor_network_params.yaml
+          path: |
+            scripts/local_testnet/logs
+          retention-days: 3
```

Comment:
Looks like these keys are invalid for this github action.
The step below defines the artifact path and retention, I think this can be removed?

Reply:
We already upload the local testnet logs, did you mean to upload the assertoor logs? They could be included in the enclave logs too but i haven't checked